test/PerformanceTests/ComponentTests/Microsoft.OData.Performance.ComponentTests.csproj

```diff
@@ -1,369 +1,40 @@
-      <HintPath>$(NugetPack)\xunit.extensibility.core.2.1.0\lib\portable-net45+win8+wp8+wpa81\xunit.core.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="xunit.execution.desktop, Version=2.1.0.3179, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
-      <HintPath>$(NugetPack)\xunit.extensibility.execution.2.1.0\lib\net45\xunit.execution.desktop.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="xunit.performance.core, Version=1.0.0.27, Culture=neutral, PublicKeyToken=67066efe964d3b03, processorArchitecture=MSIL">
-      <HintPath>..\..\..\sln\packages\Microsoft.DotNet.xunit.performance.1.0.0-alpha-build0029\lib\net46\xunit.performance.core.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="xunit.performance.execution.desktop, Version=1.0.0.27, Culture=neutral, PublicKeyToken=67066efe964d3b03, processorArchitecture=MSIL">
-      <HintPath>..\..\..\sln\packages\Microsoft.DotNet.xunit.performance.1.0.0-alpha-build0029\lib\net46\xunit.performance.execution.desktop.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
+    <None Remove="Common\Models\AdventureWorksPlus.csdl" />
+    <None Remove="Common\Models\EntityWithDifferentPropertyType.csdl" />
+    <None Remove="Common\Models\ExchangeAttachment.csdl" />
+    <None Remove="ODataReader\PayloadTemplate\Entry.json" />
+    <None Remove="ODataReader\PayloadTemplate\EntryIncludeSpatial.json" />
+    <None Remove="ODataReader\PayloadTemplate\EntryIncludeSpatialWithExpansions.json" />
+    <None Remove="ODataReader\PayloadTemplate\EntryWithExpansions.json" />
   </ItemGroup>
+
   <ItemGroup>
-    <Compile Include="Common\InMemoryMessage.cs" />
-    <Compile Include="Common\ODataMessageHelper.cs" />
-    <Compile Include="Common\ODataMessageKind.cs" />
-    <Compile Include="Common\StreamBasedRequestMessage.cs" />
-    <Compile Include="Common\StreamBasedResponseMessage.cs" />
-    <Compile Include="Common\TestContainerBuilder.cs" />
-    <Compile Include="Common\TestUtils.cs" />
-    <Compile Include="ODataReader\PayloadGenerator.cs" />
-    <Compile Include="ODataReader\ODataReaderFeedTests.cs" />
-    <Compile Include="ODataWriter\ODataWriterFeedTests.cs" />
-    <Compile Include="ODataWriter\WriteFeedPropertyTypeTests.cs" />
-    <Compile Include="Scenario\BinaryDataScaleTests.cs" />
-    <Compile Include="Scenario\DataSizeScaleTests.cs" />
-    <Compile Include="Scenario\BatchRoundTripPerfTest.cs" />
-    <Compile Include="Scenario\WriteReadFeedTestBase.cs" />
-    <Compile Include="UriParser\UriParserTests.cs" />
-    <Compile Include="Program.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Common\Models\AdventureWorksPlus.csdl">
-      <SubType>Designer</SubType>
-    </EmbeddedResource>
+    <EmbeddedResource Include="Common\Models\AdventureWorksPlus.csdl" />

```

Comment:
_Common/Models/AdventureWorksPlus.dll_ appears [here](29696ef03dad61c01ac0642cc0a54d35fda5d833c2ef3d87cdfbf3af752c3848R9) with a **Compile Action** of **None** yet it also appears here as **EmbeddedResource**. Is it deliberate or inadvertent?

Reply:
On VS it appeared as EmbeddedResource. Clicking the link you shared leads to a 404 so I'm not sure which file you're referencing.

The tests except for the files to be embedded resources cause that's where they try to read them from.

Tests call the `TestUtils.ReadTestResource` method which aemots to load the file from embedded resource.